docs/manual/common/concepts/ESAdvantage.md

```diff
@@ -0,0 +1,21 @@
+# Advantages of Event Sourcing
+
+For decades, developers have implemented persistence using the traditional Create, Read, Update, Delete (CRUD) model. As previously introduced, the event sourcing model achieves persistence by storing state changes as historical events that capture business activity before writes occur to the data store. This decouples the events from the storage mechanism, allowing them to be aggregated, or placed in a group with logical boundaries. Event Sourcing is one of the patterns that enables concurrent, distributed systems to achieve high performance, scalability and resilience.
+
+In a distributed architecture, event sourcing provides the following advantages:
+
+* In a traditional CRUD model, entity instances are usually represented dually as a mutable object in memory and a mutable row in a relational database table. This leads to the infamous "object relational impedance mismatch" (https://en.wikipedia.org/wiki/Object-relational_impedance_mismatch). Object-relational mappers were created to bridge this divide, but bring new complexities of their own. The event sourcing model treats the database as an append-only log of serialized events. It does not attempt to model the state of each entity or the relationships between them directly in the database schema. This greatly simplifies the code that writes to and reads from the database.
+
```

Comment:
I think we can review the syntax and make this a link using this instead:
` ["object relational impedance mismatch"](https://en.wikipedia.org/wiki/Object-relational_impedance_mismatch) `
(use square brackets and remove the blanskspace)